editor/js/Strings.js

```diff
@@ -6,6 +6,14 @@ function Strings( config ) {
 
 		en: {
 
+			'[prompt]/file/open': 'Any unsaved data will be lost. Are you sure?',
```

Comment:
For simplicity, I would just use `prompt/file/open`. The purpose of the brackets isn't clear without a documentation and it in general feels over-engineered, tbh.

Reply:
ok